integrations-and-supported-tools/mosaicml-composer/notebooks/Neptune_MosaicML_Composer.ipynb

```diff
@@ -0,0 +1,393 @@
+    "from composer.loggers import NeptuneLogger\n",
+    "from composer.algorithms import LabelSmoothing, ProgressiveResizing\n",
+    "from composer.callbacks import ImageVisualizer"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "xcpySPa-PsFh"
+   },
+   "source": [
+    "## Prepare dataset and dataloaders"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "uxDdtzdLPsFh"
+   },
+   "outputs": [],
+   "source": [
+    "transform = transforms.Compose([transforms.ToTensor()])\n",
+    "train_dataset = datasets.MNIST(\"data\", download=True, train=True, transform=transform)\n",
+    "eval_dataset = datasets.MNIST(\"data\", download=True, train=False, transform=transform)\n",
+    "train_dataloader = DataLoader(train_dataset, batch_size=32)\n",
+    "eval_dataloader = DataLoader(eval_dataset, batch_size=16)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "AvBNFkT0PsFh"
+   },
+   "source": [
+    "## Create `neptune_logger`\n",
+    "\n",
+    "To create a new run for tracking the metadata, you tell Neptune who you are (`api_token`) and where to send the data (`project`).\n",
+    "\n",
+    "You can use the default code cell below to create an anonymous run in the public project [common/mosaicml-composer](https://app.neptune.ai/o/common/org/mosaicml-composer). **Note**: Public projects are cleaned regularly, so anonymous runs are only stored temporarily.\n",
+    "\n",
+    "### Log to your own project instead\n",
+    "\n",
+    "Replace the code below with the following:\n",
+    "\n",
+    "```python\n",
+    "from getpass import getpass\n",
+    "\n",
+    "neptune_logger = NeptuneLogger(\n",
+    "    project=\"workspace-name/project-name\",  # replace with your own (see instructions below)\n",
+    "    api_token=getpass(\"Enter your Neptune API token: \"),\n",
+    "    tags=[\"mnist\", \"notebook\"],  # (optional) use your own\n",
+    ")\n",
+    "```\n",
+    "\n",
+    "To find your API token and full project name:\n",
+    "\n",
+    "1. [Log in to Neptune](https://app.neptune.ai/).\n",
+    "1. In the bottom-left corner, expand your user menu and select **Get your API token**.\n",
+    "1. The workspace name is displayed in the top-left corner of the app.\n",
+    "\n",
+    "    To copy the project path, in the top-right corner, open the settings menu and select **Properties**.\n",
+    "\n",
+    "For more help, see [Setting Neptune credentials](https://docs.neptune.ai/setup/setting_credentials) in the Neptune docs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "z056VXq1PsFh"
+   },
+   "outputs": [],
+   "source": [
+    "from neptune import ANONYMOUS_API_TOKEN\n",
+    "\n",
+    "neptune_logger = NeptuneLogger(\n",
+    "    api_token=ANONYMOUS_API_TOKEN,  # Replace with your own\n",
+    "    project=\"common/mosaicml-composer\",  # Replace with your own\n",
```

Comment:
We can slightly rephrase this, since it's not mandatory to replace the args.

```suggestion
    "    api_token=ANONYMOUS_API_TOKEN,  # or replace with your own (see instructions above)\n",
    "    project=\"common/mosaicml-composer\",  # or replace with your own\n",
```